tests/ModelContextProtocol.Tests/SseServerIntegrationTestFixture.cs

```diff
@@ -50,9 +48,8 @@ public SseServerIntegrationTestFixture()
     public Task<IMcpClient> ConnectMcpClientAsync(McpClientOptions? options, ILoggerFactory loggerFactory)
     {
         return McpClientFactory.CreateAsync(
-            DefaultServerConfig,
+            new SseClientTransport(DefaultServerConfig, HttpClient, loggerFactory),
```

Comment:
Nit: This could be stored in a field and disposed with the fixture. We could have done this before, and it won't really do anything since ownsHttpClient = false, but maybe it will do something at some point.

Reply:
I don't think that would be possible here since it's parameterized by a logger factory passed by the test suite.